packages/date-picker/README.md

```diff
@@ -0,0 +1,19 @@
+# Date Picker
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/date-picker.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/date-picker/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/date-picker
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/date-picker
+```
```

Comment:
need docs

Reply:
We need to decide how we want to handle README documentation going forward. What should go here vs TSDoc & Storybook?